frame/lending/src/tests/vault.rs

```diff
@@ -0,0 +1,77 @@
+use crate::{
```

Comment:
you may consider making prelude.rs

which shares most common tests used stuff as `pub use`.

so that each test `use super::prelude::*` to ease refactoring, and new test writign.